### DISCUSSION TITLE
how to get new row index after sorting

### DISCUSSION THREAD

**Discussion by AhmadAnisAkram** [TanStack/table#2547] [General]
I am making a react table which expands on clicking the checkbox but after sorting the index  of row changed so when I select all they don't expand

**Comment by gargroh** [comment]
tht will be index on final 'rows' which u r using for rendering

**Reply by AhmadAnisAkram** [reply]
HEY! thanks for your response but the problem is  after sorting the index value changes but in a row, it keeps showing the same I am using the checkbox at the header which selects all the rows as in cell we have view index but in the header, there is no view index which changes after the sorting which causes to check row which is disabled and didn't check the row which is available.

**Comment by gargroh** [comment]
Sry i didn't get it, can u open code sandbox url for the same....
